Kbs.Business.Tests/Extentions/DateTimeExtentionsTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Kbs.Business.Course;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Kbs.Business.Extentions
+{
+    public class DateTimeExtentionsTests
+    {
+        [Fact]
+        
+        public void ToDutchString_TranslatesValueDateOnly()
+        {
+            DateTime dateTime = new DateTime(2024, 6, 9);
+            var result = dateTime.ToDutchString();
+            Assert.Equal("2024-06-09", result);
+
+            dateTime = new DateTime(2024, 12, 31);
+            result = dateTime.ToDutchString();
+            Assert.Equal("2024-12-31", result);
+        }
```

Comment:
Volg de AAA standaard, samen met Theory/InlineData-attributen:

```suggestion
        public void ToDutchString_TranslatesValueDateOnly()
        {
            // Arrange
            DateTime dateTime = new DateTime(2024, 6, 9);
            
            // Act
            var result = dateTime.ToDutchString();
            
            // Assert
            Assert.Equal("2024-06-09", result);
        }
```

Kijk even af van CourseDifficultyExtensionsTests